src/main/java/com/github/lwhite1/tablesaw/io/csv/AddCellToColumnException.java

```diff
@@ -0,0 +1,53 @@
+package com.github.lwhite1.tablesaw.io.csv;
+
+import java.io.PrintStream;
+
+/**
+ * Created by Richard on 26/07/2016.
```

Comment:
Please remove auto-generated code from comments and replace with a brief description of the class